src/Umbraco.Core/Persistence/Repositories/ContentTypeBaseRepository.cs

```diff
@@ -101,7 +101,7 @@ protected IEnumerable<int> PerformGetByQuery(IQuery<PropertyType> query)
 
             var translator = new SqlTranslator<PropertyType>(sqlClause, query);
             var sql = translator.Translate()
-                                .OrderBy<PropertyTypeDto>(x => x.PropertyTypeGroupId, SqlSyntax);

```

Comment:
@clausjensen I don't quite think you know what you are doing here ;) This task is to review the relators like GroupPropertyTypeRelator to see what they are comparing on. A relator will take multiple rows and turn it into a 1:M object. For example, the GroupPropertyTypeRelator is comparing on the cmsPropertyTypeGroup.pk value, which means that this result would need to be sorted by that value in order for it to work. I'll fix it up but just want to let you know how this works.